### ISSUE TITLE
[Survival Beta - Suggestion] Navigation

### ISSUE THREAD

**Issue by Xerma** [tsunamayo/Starship-EVO#5481]
I think something we desperately need now more than ever is a way to track where things are. Ideally, it would be great to have a planetary map but most importantly, we really need the ability to create and manage waypoints to keep track of where our starter base is

**Comment by tsunamayo** [comment]
@Xerma Yeah I know that was n1 in my to do list actually, it must be written somewhere in the survival post. I will add a beacon at least, and possibly other way to add waypoint. My only issue is whether to look at them when not in a spaceship (and how if yes).

**Comment by ProPeach** [comment]
`My only issue is whether to look at them when not in a spaceship (and how if yes).`

I'd say this is necessary, but perhaps from a shorter range? 

Space Engineers tackles this with Antenna blocks, that have different signal ranges. The player has a range of a kilometer or so, with antennas on ships having larger ranges. If the signal from a beacon etc is within range, it will appear on your hud. The ranges can also stack/link, so you can pick up the signal of a ship hundreds of km away by standing near to a ship with a long range antenna etc

It also has a neat feature which merges multiple GUI icons of signals so the HUD doesn't get too messy, say if you have 3 signals on another planet it will not show each individual one, but have one marker saying "3 signals"

We'd also need a GUI to disable/enable individual markers on the HUD I suppose 🤔

**Comment by tsunamayo** [comment]
> `My only issue is whether to look at them when not in a spaceship (and how if yes).`
> 
> I'd say this is necessary, but perhaps from a shorter range?
> 
> Space Engineers tackles this with Antenna blocks, that have different signal ranges. The player has a range of a kilometer or so, with antennas on ships having larger ranges. If the signal from a beacon etc is within range, it will appear on your hud. The ranges can also stack/link, so you can pick up the signal of a ship hundreds of km away by standing near to a ship with a long range antenna etc
> 
> It also has a neat feature which merges multiple GUI icons of signals so the HUD doesn't get too messy, say if you have 3 signals on another planet it will not show each individual one, but have one marker saying "3 signals"
> 
> We'd also need a GUI to disable/enable individual markers on the HUD I suppose 🤔

Good info right here, I was also think of using the scanner for this: ie target will only display after you used your portable scanner, which display ore and target for a something like a minute or so. But I am not sure I need a portable scanner (I need a brick scanner for sure though)

**Comment by Tackietacktack** [comment]
If I understood it correctly? You want to make something like an antenna.
At Beginn we have a small antenna, then we build it a larger and get more range and then we build a small satellite dish more range.....?

**Comment by Tackietacktack** [comment]
Field glasses is an option. When we use it we get a small zoom and  information to the target. Then we know it is iron or gold and if we would be run to it.
